### PR TITLE
Add Shin Megami Tensei IV

### PR DESCRIPTION
**Pull request by robzombie91**
Adds Shin Megami Tensei IV.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by robzombie91** [comment]
[title image](https://i.imgur.com/nk0waSd.png)

**Comment by robzombie91** [comment]
Its coming out perfect on imgur but it shows the mouse when uploaded

**Comment by robzombie91** [comment]
Now there is no mouse. Im loosing my mind

File: games/shin-megami-tensei-iv/game.dat [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title = "Shin Megami Tensei IV"
+description = "Shin Megami Tensei IV is a Japanese post-apocalyptic role-playing video game developed by Atlus for the Nintendo 3DS. It is the fifth entry in the Shin Megami Tensei series, the central series of the Megami Tensei franchise, though no direct story connection exists to previous entries. It was released in May and July 2013 for Japan and North America respectively. It was released digitally in Europe in October 2014."
```
Comment (Leo626) [inline games/shin-megami-tensei-iv/game.dat]: Don't need release dates so the last two sentences aren't needed.

**Comment by robzombie91** [comment]
Couldnt find a boxart for the original release that has the esrb rating other than Pending

**Comment by Leo626** [comment]
There is [this](http://art.gametdb.com/3ds/coverHQ/US/AMXE.jpg). It's jpg so make sure to save it as png.

**Comment by robzombie91** [comment]
is the wiki entry satisfactory?

**Comment by Leo626** [comment]
I edited it to make it flow a little better.

**Pull request merged**